src/api/index.js

```diff
@@ -10,16 +10,16 @@ const SchemaValidationError = require('express-body-schema/SchemaValidationError
 
 var isTest = (process.env.NODE_ENV === 'test');
```

Comment:
The usual condition is process.env.NODE_ENV !== 'production'. Any other value than 'production' is not commonly used.

Reply:
fixed